src/loaders/DataTextureLoader.js

```diff
@@ -63,13 +63,21 @@ class DataTextureLoader extends Loader {
 
 			}
 
-			texture.wrapS = texData.wrapS !== undefined ? texData.wrapS : ClampToEdgeWrapping;
-			texture.wrapT = texData.wrapT !== undefined ? texData.wrapT : ClampToEdgeWrapping;
+			const {
+				wrapS = ClampToEdgeWrapping,
+				wrapT = ClampToEdgeWrapping,
+				magFilter = LinearFilter,
+				minFilter = LinearFilter,
+				anisotropy = 1,
+			} = texData;
```

Comment:
Apparently I have to get used to this kind of coding style^^. It is still not 100% intuitive to me...

Reply:
@mrdoob Do you think this really improves readability?

Reply:
TBH, this one has a learning cost, but if you use it for a while, you maybe like this grammar candy. After all, less code is being written. There should be no side effects.